ts/features/tos/store/selectors/index.ts

```diff
@@ -4,19 +4,19 @@ import * as O from "fp-ts/lib/Option";
 import { TosConfig } from "../../../../../definitions/content/TosConfig";
 // eslint-disable-next-line no-restricted-imports
 import { privacyUrl } from "../../../../config";
-import { backendStatusSelector } from "../../../../store/reducers/backendStatus";
+import { remoteConfigSelector } from "../../../../store/reducers/backendStatus/remoteConfig";
 
 const DEFAULT_TOS_CONFIG: TosConfig = {
   tos_url: privacyUrl,
   tos_version: 4.91
 };
 
 export const tosConfigSelector = createSelector(
-  backendStatusSelector,
-  backendStatus =>
+  remoteConfigSelector,
+  remoteConfig =>
     pipe(
-      backendStatus,
-      O.chainNullableK(bs => bs.config.tos),
+      remoteConfig,
+      O.chainNullableK(c => c.tos),
```

Comment:
Should we use `config` instead of `c`, to improve readability?

Reply:
```suggestion
      O.chainNullableK(config => config.tos),
```